package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "description": "A starter for a blog powered by Gatsby and Markdown",
```

Comment:
You should probably change all of these fields too. We can go over which ones we should change together